new/LOC6830Awesa.xml

```diff
@@ -63,10 +63,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                </note>
                 </place>
                 <listRelation>
-               <relation name="saws:formsPartOf"
+                    <relation name="dcterms:isPartOf"
                          active="LOC6830Awesa"
                          passive="LOC6829Gabal"/>
-                    <relation name="saws:isRelatedTo"
+                    <relation name="syriaca:has-relation-to-place"
```

Comment:
same as above

Reply:
same as above

Reply:
this commit was done before pull request, already corrected, other syriaca too